### PR TITLE
bugfix for --forget option

### PR DESCRIPTION
**Pull request by zacronos**

### CODE REVIEW

File: src/migrator.coffee [hunk 1]
```diff
@@ -54,7 +54,7 @@ module.exports = class Migrator
   # note: this is a static property of the class, not an instance property
   @defaults: defaults
 
-  _$.validateOptions: (optionsToCheck) ->
+  _$.validateOptions = (optionsToCheck) ->
```
Comment (zacronos) [inline src/migrator.coffee:57]: typo

File: src/migrator.coffee [hunk 2]
```diff
@@ -227,7 +227,8 @@ module.exports = class Migrator
         , 'id'
 
     .then (rowId) =>
-      @executionContext.currentMigration.rowId = parseInt(rowId[0])
+      if rowId?
+        @executionContext.currentMigration.rowId = parseInt(rowId[0])
```
Comment (zacronos) [inline src/migrator.coffee:231]: when run with --forget, rowId will be undefined

**Comment by zacronos** [comment]
@nmccready

**Comment by nmccready** [comment]
LGTM

**Pull request merged**